src/commercial-ops/delegated-authorities.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Delegated authorities
 permalink: /:path/:basename/
+last_reviewed_at: ""
```

Comment:
So I've played around with this a little

## `permalink`

Our `index.md` files have a different permalink pattern, and the `:path` and `:basename` placeholders [aren't available globally](https://jekyllrb.com/docs/permalinks/#placeholders), only on [individual pages](https://jekyllrb.com/docs/permalinks/#pages). I did manage to DRY this up in a different way though: 00c5a45

## `last_reviewed_at`

This one seems a little more challenging. In ed50b92, I've set it as the default, but that brings back the behaviour noted in the commit (and referenced in https://github.com/decaporg/decap-cms/issues/1745). When opening the page in the CMS, it prepopulates the datetime with the current time (though if you save it without manually setting it, it sets the page's front matter to an empty string), rather than blank/no value. We might have to revert to specifying the default on each page on this one. Any other ideas?

(Note the commit message is currently inaccurate, because it was working on the assumption it would just work)

Reply:
I'm out of ideas I'm afraid @yndajas. But thanks for taking on those suggestions so quickly!